modules/audio/proto/audio.proto

```diff
@@ -18,6 +18,13 @@ syntax = "proto2";
 
 package apollo.audio;
 
+enum MovingResult {
```

Comment:
might need to rename it before the onboard integration complete, "MovingResult" is not self-explanatory at all...

Reply:
Also you need "Stationary" as well

Reply:
“UNKNOWN" default assigned to 0